src/services/lnd/utils.ts

```diff
@@ -41,7 +41,7 @@ export const deleteExpiredInvoiceUser = async () => {
   date.setDate(date.getDate() - delta)
   // TODO: assert: only paid: true invoice should be remaining here
   // other invoiceUser should be deleted alongside deletion of lnd invoice
-  await InvoiceUser.deleteMany({ timestamp: { lt: date } })
```

Comment:
seems this never worked, are we sure that we want to delete all invoices (pending and paid)?